WordPress/Classes/ViewRelated/Blog/SiteManagement/SiteTagsViewController.swift

```diff
@@ -246,12 +250,60 @@ extension SiteTagsViewController {
     }
 }
 
-// MARK: - Navigation
+// MARK: - Navigation to Tag details
 extension SiteTagsViewController {
-    fileprivate func navigate(to tag: PostTag) {
-        let singleTag = SiteTagViewController(blog: blog, tag: tag)
+    fileprivate func navigate(to details: PostTag?) {
+        let data = SettingsTitleSubtitleController.Data(title: details?.name, subtitle: details?.tagDescription)
+        let confirmationStrings = confirmation()
+        let singleTag = SettingsTitleSubtitleController(data: data, confirmation: confirmationStrings)
+
+        singleTag.setAction { updatedData in
+            self.navigationController?.popViewController(animated: true)
```

Comment:
Should the references to `self` inside these closures be weak references?

Reply:
I thought so as well, but I run it through instruments and there were no apparent leaks. I will weakly them anyway, just in case